apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/dao/state/derived/Searchable.java

```diff
@@ -0,0 +1,9 @@
+package com.apollocurrency.aplwallet.apl.core.dao.state.derived;
+
+public interface Searchable {
```

Comment:
What's the point of these two interfaces Searchable and SearchableTableInterface? As I see, there is no usage of those interfaces.

Reply:
The `SearchableTableInterface` can be deprecated now, because all implementations are stubbed by throwing Exception.
The `Searchable` is used for DerivedEntity to distinguish several entities.

Reply:
Ok, 
Where is the Searchable->isSearch() method used?

Reply:
Inside BasicDbTable.java, DerivedDbTable.java

Reply:
Fixed